community-operators/etcd/etcdoperator.clusterserviceversion.yaml

```diff
@@ -2,11 +2,12 @@
 #! parse-kind: ClusterServiceVersion
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
-metadata:
+metadata:  
```

Comment:
nit: Could the trailing whitespace be removed?

Reply:
fixed